### ISSUE TITLE
The worse with hourglass than dla to train my datasets. The parameers were set as follow.

### ISSUE THREAD

**Issue by Beam-wi** [xingyizhou/CenterNet#304]
I run main.py  ctdet --exp_id component_hg --arch hourglass --batch_size 4 --master_batch 10 --lr 2.5e-4 --gpus 0 --num_epochs 1000 --lr_step 540  to train my datasets. And i change the  input resolution [512, 512] to [1024, 1024] by modifying the opts.py for the origional resolution is [5472, 3648] of my datasets.

Than i run demo.py ctdet --arch hourglass --demo E:/images/14_batch_for_test/save_data_0816/object --load_model ../models/model_best_15_epoch.pth --debug 2 to detecte. 
But the worse result with it than with dla.

By the way, the training reports as follows.
#####
epoch: 1  loss: 1.5537761919340103  hm_loss: 1.2684540771229669  wh_loss: ...
epoch: 2  loss: 0.9929959091651163  hm_loss: 0.7652857300541611  wh_loss: ...
epoch: 3  loss: 0.7778472515242721  hm_loss: 0.5751877200683507  wh_loss: ...
epoch: 4  loss: 0.6606010648563698  hm_loss: 0.4757706126062632  wh_loss: ...
epoch: 5  loss: 0.6017300094787681  hm_loss: 0.4269299381011567  wh_loss: ...
... ...
#####
It seems the  network was going on.

So could you give some advises? whether were the parameters or others wrong?

**Comment by Beam-wi** [comment]
I modifyed the initialize default_resolution of the ctdet in opts.py

**Comment by Beam-wi** [comment]
I ckecked that i couldn't change the resolution from [512, 512] to [1024, 1024].
It means i trained with [512, 512] input size.  Even so , the result should not so bad. (can't detecte 10 from 100)
I wonder what was wrong from the set. 
By he way, could you provide how to train the less hourglass it is to large for engineering application?

**Comment by Beam-wi** [comment]
Now, i set the batch size to 1, and the input resolution is [1024, 1024]. The network is working on. 
Could you give me some advices to set the main.py and demo.py?
The Curent settings for main.py like that:
main.py ctdet --exp_id component_hg --arch hourglass --batch_size 1 --master_batch 1 --lr 2.5e-4 --gpus 0 --num_epochs 100 --lr_step 120

**Comment by xingyizhou** [comment]
I didn't understand the question. Please try to shorten the question into one sentence.

**Comment by Ai-is-light** [comment]
python main.py ctdet --exp_id component_hg --arch hourglass --batch_size 4 --master_batch 10 --lr 2.5e-4 --gpus 0 --num_epochs 1000 --lr_step 540
usage: main.py [-h] [--task TASK] [--dataset DATASET] [--exp_id EXP_ID]
               [--test] [--debug DEBUG] [--demo DEMO]
               [--save_inference_ouput_path SAVE_INFERENCE_OUPUT_PATH]
               [--load_model LOAD_MODEL] [--resume]
               [--model_save_path MODEL_SAVE_PATH] [--save_name SAVE_NAME]
               [--data_stage DATA_STAGE] [--gpus GPUS]
               [--num_workers NUM_WORKERS] [--not_cuda_benchmark]
               [--seed SEED] [--print_iter PRINT_ITER] [--hide_data_time]
               [--save_all] [--metric METRIC] [--vis_thresh VIS_THRESH]
               [--debugger_theme {white,black}] [--arch ARCH]
               [--head_conv HEAD_CONV] [--down_ratio DOWN_RATIO]
               [--input_res INPUT_RES] [--input_h INPUT_H] [--input_w INPUT_W]
               [--lr LR] [--lr_step LR_STEP] [--num_epochs NUM_EPOCHS]
               [--batch_size BATCH_SIZE]
               [--master_batch_size MASTER_BATCH_SIZE] [--num_iters NUM_ITERS]
               [--val_intervals VAL_INTERVALS] [--trainval] [--flip_test]
               [--test_scales TEST_SCALES] [--nms] [--K K]
               [--not_prefetch_test] [--fix_res] [--keep_res]
               [--not_rand_crop] [--shift SHIFT] [--scale SCALE]
               [--rotate ROTATE] [--flip FLIP] [--no_color_aug]
               [--aug_rot AUG_ROT] [--aug_ddd AUG_DDD] [--rect_mask]
               [--kitti_split KITTI_SPLIT] [--mse_loss] [--reg_loss REG_LOSS]
               [--hm_weight HM_WEIGHT] [--off_weight OFF_WEIGHT]
               [--wh_weight WH_WEIGHT] [--hp_weight HP_WEIGHT]
               [--hm_hp_weight HM_HP_WEIGHT] [--dep_weight DEP_WEIGHT]
               [--dim_weight DIM_WEIGHT] [--rot_weight ROT_WEIGHT]
               [--peak_thresh PEAK_THRESH] [--norm_wh] [--dense_wh]
               [--cat_spec_wh] [--not_reg_offset] [--agnostic_ex]
               [--scores_thresh SCORES_THRESH] [--center_thresh CENTER_THRESH]
               [--aggr_weight AGGR_WEIGHT] [--dense_hp] [--not_hm_hp]
               [--not_reg_hp_offset] [--not_reg_bbox] [--eval_oracle_hm]
               [--eval_oracle_wh] [--eval_oracle_offset] [--eval_oracle_kps]
               [--eval_oracle_hmhp] [--eval_oracle_hp_offset]
               [--eval_oracle_dep]
main.py: error: unrecognized arguments: ctdet

**Comment by Ai-is-light** [comment]
what's wrong?

**Comment by sisrfeng** [comment]
try 
--task ctdet
 instead of 
ctdet
